src/components/app/content/cardList/WithLoadingDelay.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import "./index.css"
+
+export default function WithLoadingDelay(Card) {
+    return ({isLoading, ...props}) => {
+        if (isLoading) {
+            return (<div className="card card-layout lds-container" key={props.key}>
+                <div className="lds-default">
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                    <div/>
+                </div>
+            </div>)
```

Comment:
I understand that this is the custom spinner implementation, but looks quite odd. Far more easier to use some package like `react-spinners` or whatever. OR, if you really want to use that solution - at least move it to a separate custom component.